app/screens/settings/config.ts

```diff
@@ -1,7 +1,10 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {SNACK_BAR_TYPE} from '@constants/snack_bar';
```

Comment:
0/5 the content of this file seems a bit out of place (I cannot wrap my head around it being in the screens folder, with a file name of config). Not a big deal, and can stay here, but I wanted to flag it.